### DISCUSSION TITLE
"Blacklisting" dates for date-extraction

### DISCUSSION THREAD

**Discussion by Roluf** [jonaswinkler/paperless-ng#1349] [Feature Requests]
Hi,

an option to "blacklist" dates (eg. birthday) for automatic date-extraction would be great. So if the worker finds that date first in the document it takes the next it can find.

Thanks for your great work!

**Comment by TheFehr** [comment]
The [docs](https://paperless-ng.readthedocs.io/en/latest/configuration.html?highlight=ignore#software-tweaks) contain an explaination for PAPERLESS_IGNORE_DATES which I'd guess is what you are looking for

**Reply by Roluf** [reply]
Oh, cool. Totally missed that, it's exactly what i was looking for 👍
Thank you!

**Reply by fdw** [reply]
I would like to use this opportunity to advertise https://github.com/jonaswinkler/paperless-ng/issues/781 ;)
